rfcs/070-concepts-api-changes/README.md

```diff
@@ -0,0 +1,324 @@
+
+And these source concepts are linked to a Wikidata source concept:
+
+| Source concept ID | Source   | Linked to                  | Label      | Description                                                                                                |
+|:------------------|----------|----------------------------|------------|------------------------------------------------------------------------------------------------------------|
+| Q949149           | Wikidata | `D012499` and `sh85117296` | Sanitation | public health conditions related to clean drinking water and adequate disposal of human excreta and sewage |
+
+The source identifiers of all of these source concepts will be stored together in Elasticsearch, and they will all be returned
+from the API in a single list.
+
+Next, we can use a combination of `NARROWER_THAN` and `SAME_AS` edges between source concepts, and `HAS_SOURCE_CONCEPT` edges
+to obtain a list of catalogue concepts which are broader than the four catalogue concepts listed above:
+
+| Concept IDs                            | Label                        |
+|:---------------------------------------|------------------------------|
+| sagccnc9, z44pyycv                     | Environmental Health         |
+| c9ayxjtj, uj4hz4ct, zvpgcgjv, hfhs58ra | Public health                |
+
+In a similar way, we can obtain a list of related concepts by utilising `RELATED_TO`/`HAS_PARENT` edges between source concepts:
+
+| Concept IDs                            | Label                        |
+|:---------------------------------------|------------------------------|
+| g7nnn9wn, dwqd7gvs, xtndzsda           | Environmental Policy         |
+| z7azx5qx, mwyumfq2, sjxv6uys           | Hygiene                      |
+| yz3xs9c9, cqpmracr                     | Communicable Disease Control |
+| zc9y7m45, kx79h6jm, xcdy58xp, mz9syutk | Sanitary Engineering         |
+
+Putting everything together, we can construct a full JSON response returned to the front-end:
+
+```
+GET /concepts/eva7r2dw
+{
+  // The Wellcome IDs of all 'merged' concepts are included
+  "id": ["eva7r2dw", "d8z89dv6", "mwyumfq2", "jubdg55b"],
+  
+  // The identifiers of all source concepts are included
+  "identifiers": [
+    // Notice that there are two LoC identifiers. This is because LoC sometimes stores duplicate concepts.
+    {
+      "identifierType": "lc-subjects",
+      "value": "sh85117296",
+      "type": "Identifier"
+    },
+    {
+      "identifierType": "lc-subjects",
+      "value": "sh00007929",
+      "type": "Identifier"
+    },    
+    {
+      "identifierType": "nlm-mesh",
+      "value": "D012499",
+      "type": "Identifier"
+    },
+    {
+      "identifierType": "wikidata",
+      "value": "Q949149",
+      "type": "Identifier"
+    }
+  ],
+   
+  // The MeSH label takes priority over any other labels
```

Comment:
If we do not merge concepts into one document, I would still expect us to try to enhance concept pages by merging fields from matched concepts, using a hierarchy as we do in the works merger.